test/distributed/fsdp/test_fsdp_param_exec_order_wrap.py

```diff
@@ -25,25 +28,31 @@ def __init__(self) -> None:
         )
         self.relu = torch.nn.ReLU()
 
-    def forward(self, x):
+    def forward(self, x: Any, flag: bool = True):
         # `layer0` -> `layer2` -> `layer1`
         # the forward execution order is NOT consistent with the model definition order.
         z = self.relu(self.layer0(x))
         z = self.relu(self.layer2(z))
-        z = self.relu(self.layer1(z))
+        if flag:
+            z = self.relu(self.layer1(z))
         return z
 
     def get_input(self, device: torch.device):
-        return (torch.randn((8, 6)).to(device), )
+        return (torch.randn((8, 6)).to(device),)
 
     def get_loss(self, input, output):
         return (output - input[0]).sum()
 
     @staticmethod
-    def wrap(sharding_strategy: ShardingStrategy, device: torch.device, init_policy=always_wrap_policy):
+    def wrap(
+        sharding_strategy: ShardingStrategy,
+        device: torch.device,
+        wrap_policy=always_wrap_policy,
```

Comment:
nit: Do we want this default argument? Do we ever rely on it being the default? I see several usages where we pass in the `ParamExecOrderWrapPolicy`.

Reply:
removed the default arg